environments.json

```diff
@@ -1496,15 +1496,17 @@
     "family": "V8",
     "short": "CH&nbsp;61,<br>OP&nbsp;48",
     "release": "2017-09-05",
+    "retire": "2017-10-17",
     "note_id": "experimental-flag",
-    "obsolete": false
+    "obsolete": true
```

Comment:
I'm really not comfortable with v61 becoming "obsolete" merely because v62 came out *today*. I'd expect "obsolete" to be, perhaps, 2 versions back at least.

Reply:
That would cause more Chrome versions to be shown in the table, wouldn't it?

If anything gets changed in that regard, this should apply to Firefox as well.

Reply:
Yes, I'd definitely expect, for all browsers, that "a new version was released today" is not enough to make "yesterday's newest version" obsolete.

Reply:
OK. This sounds reasonable to me. Let's make Firefox changes in another PR

Reply:
I disagree.  I DO believe that a new version of Chrome and Firefox makes the previous version obsolete, since the previous version is discontinued and stops receiving security updates.  In Chrome's case, the previous version is not even available for installation anymore.

Reply:
Safari 10.1 won't receive further updates either and it's not marked as obsolete. So upstream support isn't treated as a hard requirement for not being obsolete in the data. The same applies to iOS.

Reply:
Showing last two versions makes a lot of sense to me. Browser update doesn't happen immediately.

Reply:
Agreed; obsolescence has nothing to do with support status imo.